frontend/src/components/gestionnaireComponents/ReportedProblems.tsx

```diff
@@ -0,0 +1,77 @@
+import React, {useEffect, useState} from "react";
+import {
+    putGetUnresolvedProblems,
+    putResolveProblem
+} from "../../services/gestionnaireServices/GestionnaireFetchService";
+import IUser from "../../models/IUser";
+import {Button, Col, Table} from "react-bootstrap";
+import {IReportedProblem} from "../../models/IReportedProblem";
+
+const ReportedProblems = ({user: connectedUser}: { user: IUser }) => {
+    const [problems, setProblems] = useState<IReportedProblem[]>([]);
+    const resolve = async (id: number, index: number) => {
+        await putResolveProblem(connectedUser.token, id);
+        setProblems(problems.splice(index + 1, 1));
+    }
+    useEffect(() => {
+        const fetchProblems = async () => {
+            const response = await putGetUnresolvedProblems(connectedUser.token)
+            if (response.ok) {
+                const data = await response.json();
+                setProblems(data);
+            }
+        }
+        fetchProblems()
+    }, [connectedUser]);
+
+    const getBody = () => {
+        if (problems.length === 0) {
+            return <h1 className="text-light text-center">Aucun problèmes signalés</h1>
```

Comment:
I still keep the table in all the other tables if thing is zero and add a row saying this instead